test/integration/query-interface.test.js

```diff
@@ -112,6 +113,114 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         expect(tableNames).to.include('my_test_table2');
       });
     }
+
+    if (dialect.supports.schemas) {
+      describe('schema support tests', () => {
```

Comment:
To reduce nesting:

```suggestion
    describe('schema support tests', () => {
      if (dialect.supports.schemas) {
        return;
      }
```

Reply:
https://github.com/sequelize/sequelize/pull/15101/commits/1a800b7a2f2d461fe8c07029e815e98c3b0b938b

Reply:
There was a mistake in the suggestion, what she meant was;
```suggestion
    if (!dialect.supports.schemas) {
      return;
    }
```
This way the three tests don't have to be nested under the if-statement

Reply:
thank you, fixed it here

https://github.com/sequelize/sequelize/pull/15101/commits/25707a35420e5db1753ebb701304df5f17dafb9a